packages/frontend-core/src/components/grid/cells/RelationshipCell.svelte

```diff
@@ -322,6 +351,21 @@
   </GridPopover>
 {/if}
 
+{#if relationshipFields && !focused}
+  <GridPopover {open} {anchor}>
+    <div class="relationship-fields">
+      {#each Object.entries(relationshipFields) as [fieldName, fieldValue]}
+        <div class="relationship-field-name">
```

Comment:
NAB: I think it might look nicer with a different font style for column names. Something like all caps, grey-600 or gery-700, slightly smaller font size (to balance the caps).

Reply:
Good shout, it looks much nicer!